server/applicationset/applicationset.go

```diff
@@ -148,6 +169,108 @@ func (s *Server) List(ctx context.Context, q *applicationset.ApplicationSetListQ
 
 }
 
+func (s *Server) Watch(q *applicationset.ApplicationSetWatchQuery, ws applicationset.ApplicationSetService_WatchServer) error {
```

Comment:
Please consider adding docs to all public symbols (functions, methods, constants, etc)